src/tactic/ui/tools/repo_browser_wdg.py

```diff
@@ -3696,16 +3716,10 @@ def get_display(my):
             allow_drag = False
 
         # FIXME: This kwarg does not take effect
-        # TODO: Drag and drop upload
+        upload_mode = False
```

Comment:
Upload mode is for tile layout. It allows users to check in images by dragging on top of a tile. I added this comment because setting upload_mode = False for tile layout didn't disable the upload behavior.